frontend/src/hooks/useFetchThemes.ts

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react';
+import axios from 'axios';
+import { THEME_URL } from '@/constants/url.ts';
+
+interface Theme {
+  themeId: number;
+  name: string;
+  image: string;
+}
+
+export default function useFetchThemes() {
+  const [themes, setThemes] = useState<Theme[]>([]);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [error, setError] = useState<boolean>(false);
+
+  useEffect(() => {
+    const fetchThemes = async() => {
+      try {
+        const res = await axios.get(THEME_URL);
```

Comment:
any로 읽히지 않게 데이터 타입을 넣어주면 좋아보여요 😊

```suggestion
        const res = await axios.get<{data: Theme[]}>(THEME_URL);
```

Reply:
<{ data: T[] }>를 추가하여 각 api 맞는 타입 오도록 설정하였습니다.